src/screens/UserPortal/Chat/Chat.tsx

```
@@ -137,154 +146,156 @@ export default function chat(): JSX.Element {
-          toggleCreateDirectChatModal={toggleCreateDirectChatModal}
-          createDirectChatModalisOpen={createDirectChatModalisOpen}
-          chatsListRefetch={chatsListRefetch}
-          chats={chats}
-        ></CreateDirectChat>
-      )}
+        {createGroupChatModalisOpen && (
+          <CreateGroupChat
+            toggleCreateGroupChatModal={toggleCreateGroupChatModal}
+            createGroupChatModalisOpen={createGroupChatModalisOpen}
+            chatsListRefetch={chatsListRefetch}
+          ></CreateGroupChat>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)